class.jetpack-jitm.php

```diff
@@ -414,6 +426,62 @@ function videopress_media_upload_warning_msg() {
 		<?php
 	}
 
+	/**
+	 * Display message prompting user to install WooCommerce Services.
+	 *
+	 * @since 4.6
+	 */
+	function woocommerce_services_msg() {
+		if ( ! current_user_can( 'manage_woocommerce' ) ) {
```

Comment:
We should also check for the `install_plugins` capability here.  In a multisite environment, non-super-admins can't install plugins.